src/main/java/de/tum/in/www1/artemis/web/rest/StudentQuestionResource.java

```diff
@@ -200,6 +205,28 @@ public StudentQuestionResource(StudentQuestionRepository studentQuestionReposito
         return new ResponseEntity<>(studentQuestions, null, HttpStatus.OK);
     }
 
+    /**
+     *
+     * GET /courses/{courseId}/student-questions : get all student questions for course.
+     * @param courseId the course that the student questions belong to
+     * @return the ResponseEntity with status 200 (OK) and with body all student questions for exercise
```

Comment:
Same here.

Reply:
I changed it to say `course` instead of exercise now.